andela/ah-backend-iroquois
# #159952003 Reset password via email

#### What does this PR do?
  - Enable user to reset the password via email

#### Description of the task to be completed
 - The user invokes a password reset email by passing in their email.
 - A user receives a password reset email. 
 - The user supplies a new password.
    
#### How should this be manually tested?
1.  This can be manually (locally) tested by using this URL 
     http://127.0.0.1:8000/api/users/reset/password/ with a POST  method in Postman.

2. You then define your JSON body as below.

               {"user": {
                           "email": "lutaaya2@gmail.com"
                  }
          
            }

 3.   Check your email for a link. 
 4.   Copy the link and paste it in Postman URL. Ensure that method is PATCH. 
         http://127.0.0.1:8000/api/user/reset-password/<token>.
      

  5.  Then input the new password in the body of Postman and then submit.

     {"user":  {
                    "password": "new_password"
          }
          
}

#### What are the relevant pivotal tracker stories?
     #159952003
